common/math/spline_interpolation/src/spline_interpolation.cpp

```diff
@@ -163,12 +163,7 @@ std::vector<double> SplineInterpolator::solveLinearSystemExplicit()
     w.push_back(tmp);
   }
 
-  for (size_t i = N - 2; i > 0; --i) ans[i] = ans[i] - ans[i + 1] * w[i];
-
-  return ans;
-}
-
-void LinearSystemSolver::setCoefficients(
+  for (size_t i = N - 2; i > 0; --i) ans[  <buildtool_depend>catkin</buildtool_depend>
```

Comment:
Got something here pasted mistakenly.

Reply:
sorted, oddly enough it compiled fine so must've copied just before committing